src/ir/comp.rs

```diff
@@ -1078,8 +1078,13 @@ impl CompInfo {
             return None;
         }
 
+        if self.kind == CompKind::Union && self.fields().len() == 0 {
+            return None;
+        }
+
         let mut max_size = 0;
-        let mut max_align = 0;
```

Comment:
Instead, we should probably return `None` if we didn't find _any_ of the field layouts, but this is fine too I think.

Reply:
returning `None` if none the field fields have a layout cause theses tests to fail internally with  'Unable to get layout information?', so leaving this as-is for now.

header_issue_493_1_0_hpp
header_issue_493_hpp
header_transform_op_hpp

Reply:
Sounds good, thanks :)